docs/Guide/CLI/getting-started.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Getting Started
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
+
+## Installation
+
+```bash npm2yarn2pnpm
+npm install @sapphire/cli -g
+```
+
+:::caution
+
+You can install the CLI as a devDependency, but we recommend installing it globally.
+
+:::
+
+<details>
+<summary>New project</summary>
+
+## Creating a bot
+
+```bash
+sapphire new
+```
+
+Run this command to start.
```

Comment:
This is roughly the equivalent of top-posting in emails. Please put instructions *before* the demonstrations to be consistent and to preserve the logical flow of a guide.

If it helps, try to remember how research is done: method first, practical application second, comments and further reading last.

Reply:
@enxg

Reply:
Done